site/en/docs/extensions/mv3/getstarted/tut-tabs-manager/index.md

```diff
@@ -0,0 +1,382 @@
+    await chrome.tabs.update(tab.id, { active: true });
+    await chrome.windows.update(tab.windowId, { focused: true });
+  });
+
+  elements.add(element);
+}
+document.querySelector("ul").append(...elements);
+...
+```
+
+{% Details %}
+{% DetailsSummary %}
+💡 **Interesting JavaScript used in this code**
+{% endDetailsSummary %}
+
+- The [Collator](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Intl/Collator) is used to sort the tabs array by title in any language.
+- The [template tag](https://web.dev/webcomponents-template/) is used to
+  define an HTML element that can be cloned instead of using `document.createElement()` to create each item.
+- The [URL constructor](https://developer.mozilla.org/docs/Web/API/URL/URL) is used to create and parse URLs.
+
+{% endDetails %}
+
+#### Group the tabs {: #group}
+
+Add the following code to create a button that will group all the tabs using [`tabs.group()`]() and
+move them into the current window.
+
+{% Label %}popup.js:{% endLabel %}
+
+```js
+...
+const button = document.querySelector("button");
+button.addEventListener("click", async () => {
+  const group = await chrome.tabs.group({ tabIds: tabs.map(({ id }) => id) });
+  await chrome.tabGroups.update(group, { title: "DOCS" });
+});
+```
+
+The [TabGroups][api-tabgroups] API allows the extension to name the group and choose a background
+color. Add the `“tabGroups”` permission to the manifest by adding the following code:
+
+{% Label %}manifest.json:{% endLabel %}
+
+```json
+{
+  ...
+  "permissions": [
+    "tabGroups"
+  ]
+}
+```
+
+## Test that it works {: #try-out }
+
+Verify that the file structure of your project looks like the following: 
+
+```text
+└── Tabs-manager/
+    ├── manifest.json
+    ├── popup.js
+    ├── popup.css
+    ├── popup.html
+    └── images/
+        ├── icon-16.png
+        ├── icon-32.png
+        ├── icon-48.png
+        └── icon-128.png
+```
+
+### Load your extension locally {: #locally }
+
+To load an unpacked extension in developer mode, follow the steps in [Development
+Basics][doc-dev-basics-unpacked].
+
+### Open a few documentation pages {: #open-sites}
+
+Open the following docs in different windows:
+
+- [Design the user interface][doc-ui]
+- [Discovery on the Chrome Web Store][cws-discovery]
+- [Extension development overview][doc-devguide]
+- [Manifest file format][doc-manifest]
+- [Publish in the Chrome Web Store][cws-publish]
+- [Welcome to Chrome Extension Development][doc-welcome]
+
+
+Click on the popup. It should look like this:
+
+<figure>
+{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/8q5ps3dqw4p2BOZRpIJT.png", 
+alt="Tabs Manager extension popup", width="600", height="230", class="screenshot" %}
+  <figcaption>
+  Tabs Manager extension popup
+  </figcaption>
+</figure>
+
+## Potential enhancements {: #challenge }
+
+Based on what you’ve learned today, try to implement any of the following:
+
+- Improve the popup stylesheet.
```

Comment:
Nit: Slightly less onerous wording. "Improve" may feel like it's too hard to do, but something like "customize" or "tweak" may feel more playful and inviting.

```suggestion
- Customize the popup stylesheet.
```